hackathon_site/dashboard/frontend/src/api/orders.ts

```diff
@@ -0,0 +1,17 @@
+import SubmittedIcon from "../assets/images/icons/statusIcons/unfulfilled-status.svg";
```

Comment:
Regarding the imports if you notice you don't see the `../` in our imports since they are absolute(from certain directories like src, dashboard, frontend etc). So could you change this to 'assets/images/...' and so on for all your imports. 
![image](https://github.com/ieeeuoft/hackathon-template/assets/43456930/a526c199-6d78-4a85-ab51-c46dc7d41a3b)